about.html

```diff
@@ -22,24 +21,19 @@
 <h3>Applications</h3>
 
 <ul>
-<li>PcManFm-qt - File Manager</li>
-<li>Lximage-qt - Image Viewer</li>
-<li>QTerminal - Terminal Emulator</li>
-<li>Qps - Process Viewer</li>
-<li>Screengrab - Screen Capturer</li>
-<li>LXQt-archiver - Archive Manager</li>
-<li>LXQt-runner - Application launcher and calculator</li>
+<li>lxqt-runner — Launch applications and do calculations</li>
+<li>PCManFM-Qt — Browse files</li>
+<li>LXImage-Qt — View images</li>
+<li>QTerminal — Use the terminal</li>
+<li>ScreenGrab — Capture screenshots</li>
+<li>LXQt Archiver — Manage archives</li>
+<li>Qps — Manage processes</li>
 </ul>
 <br/>
 
-<p>LXQt is available for all major Linux and BSD distributions so you may just try it
-out on your system or in a VM.<br>
-More information about installing can be found in the
-<a href="https://github.com/lxqt/lxqt/wiki/">LXQt GitHub Wiki</a>.
+<p>Try LXQt in most Linux or BSD distributions.<br>
+Learn to install it on the <a href="https://github.com/lxqt/lxqt/wiki/">LXQt wiki</a>.
 </p>
 <p>
-Historically, LXQt was the product of the merge between LXDE-Qt, an initial Qt
-flavour of LXDE, and Razor-qt, a project aiming to develop a Qt based desktop environment.<br>
-More on this topic can be found in the
```

Comment:
Does this make anyone want to read more about the topic?

Reply:
It gives a short basic history info, the rest depends on the reader.

Reply:
The reader may not know what "merge" is. (As opposed to "merger".)
"Qt based desktop environment." should be "Qt-based desktop environment."
What is "LXDE".
Why is anyone interested in the topic of groups merging or what they aimed to do?
That is all there is to it, and more.
Anyone in the know will already know as much.